src/entity/user/Admin.java

```diff
@@ -0,0 +1,152 @@
+        System.out.println("Complete the following information in order to register!");
+        System.out.println("First Name: ");
+        String firstName = scanner.nextLine();
+        System.out.println("Last Name: ");
+        String lastName = scanner.nextLine();
+        System.out.println("If you register with ID => type 1");
+        System.out.println("If you register with Passport => type 2");
+        String userDocumentNr = scanner.nextLine();
+        UserDocument userDocument;
+
+        switch (userDocumentNr){
+            case ("1"):
+                userDocument = UserDocument.ID;
+                break;
+            case ("2"):
+                userDocument = UserDocument.PASSPORT;
+                break;
+            default:
+                System.out.println("Wrong input for userDocument");
+                //userDocument = UserDocument.ID;
+                return;
+        }
+
+        System.out.println("Address: ");
+        String address = scanner.nextLine();
+        System.out.println("Telephone: ");
+        String telephone = scanner.nextLine();
+        System.out.println("Username: ");
+        String username = scanner.nextLine();
+        System.out.println("Password: ");
+        String password = scanner.nextLine();
+        System.out.println("Email: ");
+        String email = scanner.nextLine();
+
+        Customer customer = new Customer(firstName, lastName, userDocument, address, telephone, username, password, email);
+        hotel.getCustomerList().add(customer);
+    }
+
+    public void viewAllBookings(){ System.out.println(hotel.getBookingList()); }
+
+    public void viewAllRooms(){
+        RoomTypeComparator roomTypeComparator = new RoomTypeComparator();
+        hotel.getRoomList().sort(roomTypeComparator);
+        System.out.println(hotel.getRoomList());
+    }
+
+    public void viewAllPayments(){ System.out.println(hotel.getPaymentList()); }
+
+    public void viewAllCustomers(){
+        System.out.println(hotel.getCustomerList());
+    }
+
+    public void changeRoomStatus(int roomNumber, RoomStatus roomStatus){
+        for (Room room :hotel.getRoomList()){
+            if (room.getRoomNumber() == roomNumber && room.getRoomStatus() != roomStatus){
+                room.setRoomStatus(roomStatus);
+            }
+        }
+    }
+
+    public void changeRoomType(int roomNumber, RoomType roomType){
+        for (Room room :hotel.getRoomList()){
+            if (room.getRoomNumber() == roomNumber && room.getRoomType() != roomType){
+                room.setRoomType(roomType);
+            }
+        }
+    }
+
+    public void deleteRoom(int roomNumber){
+        // TODO: verificare daca aceasta camera are booking-uri asociate => in viitor => schimbi respectivul booking, daca este posibil
+        //                                                               => in prezent => nu poti sterge camera
+        hotel.getRoomList().removeIf(x-> x.getRoomNumber() == roomNumber);
+    }
```

Comment:
all these methods should be in the service for Admin - AdminService, in entities we have just fields, setters, getters, constructors, toString, equals, hashCode